Jenkinsfile

```diff
@@ -1,23 +1,60 @@
 #!groovy
 
-node {
-  checkout scm
 
-  def dockerRepoName = 'zooniverse/front-end-monorepo'
-  def dockerImageName = "${dockerRepoName}:${env.BUILD_ID}"
-  def newImage = null
+timeout(20) {
```

Comment:
Do we need any handling in case of timeout? I suppose jenkins will just report a failure?

Reply:
Yeah, it'll just fail.